cfme/common/provider.py

```diff
@@ -540,7 +540,7 @@ def delete(self, cancel=True):
             cancel: Whether to cancel the deletion, defaults to True
         """
         view = navigate_to(self, 'Details')
-        item_title = 'Remove this {} Provider from Inventory'
+        item_title = 'Remove this Infrastructure Provider'
```

Comment:
I think this will only work for an infrastructure provider, but will fail for other providers. 
`item_title = 'Remove this {} Provider'` will work for all providers.

Reply:
Good point